code/game/objects/structures/displaycase.dm

```diff
@@ -331,33 +320,40 @@
 			else
 				to_chat(user, "You are too far to set the plaque's text.")
 
-		SSpersistence.SaveTrophy(src)
+		if(scanned_for_replication == FALSE)
+			var/scan = alert(user, "Scan this item for replication in future shifts? You can only do this once.", "Scan for replication?", "No", "Yes")

```

Comment:
Add a variable that checks if the display is currently in use and stop the interaction when it's set. Set it here or at the start of the whole chain.